new/PRS14031John.xml

```diff
@@ -53,12 +53,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <faith type="Coptic"/>
                     <occupation type="ecclesiastic">Bishop of Nikiu</occupation>
                     <nationality type="Egypt"/>
-                    <birth>ca. 540</birth>
-                    <death>610 to 620</death>
+                    <birth when="0540" cert="medium"/>
+                    <death atLeast="0610" atMost="0620"/>
```

Comment:
we use notBefore and notAfter

Reply:
I changed the date of death last week and push it. Do you mean, I should change the date of birth to notBefore and notAfter as well, giving an approximate range instead of a point?